sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs

```diff
@@ -39,11 +39,11 @@ public void ResetDiagnostics()
 
         #region Batch Mechanics
         [Test]
-        public void Batch_EmptyFails()
+        public async Task Batch_EmptyFails()
```

Comment:
Should add Async to the name of the Tests that become async

Reply:
All of the tests run in both sync and async. Adding Async to each test is a bit misleading.